<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWorkWindow</class>
 <widget class="QDialog" name="StartWorkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ввод в работу</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Документ</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeSpec::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDoc">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список изделий для ввода в работу</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwProduct">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>531</width>
     <height>261</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbProdToWork">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести в работу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbOrganization">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Организация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>StartWorkWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
